requirements.txt

```diff
@@ -13,7 +13,7 @@ expandvars==0.9.0
 fastapi[all]==0.89.1
 fastapi-caching[redis]==0.3.0
 fastapi-pagination[sqlalchemy]==0.11.4
-fideslang==1.4.5
+fideslang==1.4.6a0
```

Comment:
I am back to pinning to a specific fideslang commit here that is based off of 1.4.6, instead of 1.4.5. I am hoping tomorrow we get the fideslang 2.0 work in, and I can just move forward on getting the proper release instead of bothering to make an alpha tag. we are close here!

Reply:
now pointed to the correct fideslang version!